Wikipedia/Code/WMFArticleFooterMenuViewController.m

```diff
@@ -39,6 +39,16 @@ - (void)viewWillAppear:(BOOL)animated {
     [self.tableView deselectRowAtIndexPath:[self.tableView indexPathForSelectedRow] animated:YES];
 }
 
+- (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSInteger)section {
```

Comment:
@bgerstle 
With grouped tables we don't have to have any special logic to have separator on top of the first cell and on bottom of the last cell.